Sprint-1/errors/0.js

```diff
@@ -1,2 +1,3 @@
-This is just an instruction for the first activity - but it is just for human consumption
-We don't want the computer to run these 2 lines - how can we solve this problem?
\ No newline at end of file
+/*This is just an instruction for the first activity - but it is just for human consumption
+We don't want the computer to run these 2 lines - how can we solve this problem? 
+because these sentences were not nested inside JS's comment syntax.*/
```

Comment:
There is another comment syntax which is easier to use when there is only a few lines of text to comment.